jobs/rocoto/wdqms.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
So all of the other rocoto job scripts in `dev/gfs.v16` are still ksh:
```
#!/bin/ksh -x
```
As much as I'd prefer the whole v16 package was converted to bash...perhaps for consistency make this ksh like the others? We do have a mix of shells across the v16 system though, so I won't push hard for this. Wanted to at least note the difference.

Reply:
ok. will update momentarily.